src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { Todo } from '../types/Todo';
+import { Filters } from '../types/Filters';
+import cn from 'classnames';
+
+type Props = {
+  currentFilter: Filters;
+  setCurrentFilter: Dispatch<SetStateAction<Filters>>;
+  todos: Todo[];
+  handleClearCompleted: () => Promise<void>;
+};
+
+const filters = [
```

Comment:
you can use only filter types from the enum Filters for mapping without creating a preliminary object

Reply:
Good point, fix this